app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,4 @@
+class WelcomeController < ApplicationController
+  def index
```

Comment:
I remember there might be a way to condense methods that don't have anything in them to one line.